package.json

```
@@ -126,9 +126,9 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.10.2",
-    "@carbon/charts": "^0.36.2",
+    "@carbon/charts": "^0.40.8",
     "@carbon/charts-react": "^0.36.2",
-    "@carbon/colors": "10.8.0",
+    "@carbon/colors": "^10.15.0",
```

This should be pinned, not caret range.
```suggestion
    "@carbon/colors": "10.15.0",
```